pkg/kapp/config/default.go

```diff
@@ -643,6 +643,8 @@ changeRuleBindings:
               - anyMatcher: {matchers: *rbacMatchers}
               - anyMatcher: {matchers: *podRelatedMatchers}
       - hasNamespaceMatcher: {}
+
+StripNameHashSuffix: {}
```

Comment:
`StripNameHashSuffix` -> `nameHashSuffix` OR `stripNameHashSuffix`
Also, I think that we should skip adding an empty config to default config.